BookingsApi/BookingsApi/Controllers/WorkHoursController.cs

```diff
@@ -16,36 +15,36 @@ namespace BookingsApi.Controllers
     [Produces("application/json")]
     [Route("workallocation")]
     [ApiController]
-    public class WorkAllocationController : Controller
+    public class WorkHoursController : Controller
     {
         private readonly ICommandHandler _commandHandler;
 
-        public WorkAllocationController(ICommandHandler commandHandler)
+        public WorkHoursController(ICommandHandler commandHandler)
         {
             _commandHandler = commandHandler;
         }
 
         /// <summary>
-        /// Save vho work schedule
+        /// Save vho work hours
         /// </summary>
-        /// <param name="uploadWorkAllocationRequests"></param>
+        /// <param name="uploadWorkHoursRequests"></param>
         /// <returns>List of usernames that were not found</returns>
-        [HttpPost("SaveWorkAllocations")]
-        [OpenApiOperation("SaveWorkAllocations")]
+        [HttpPost("SaveWorkHours")]
+        [OpenApiOperation("SaveWorkHours")]
         [ProducesResponseType(typeof(List<string>), (int) HttpStatusCode.OK)]
         [AllowAnonymous]
```

Comment:
Need to remove this AllowAnonymous

Reply:
also makes me wonder if we should add an `[Authorize(AppRoles.VhTeamLead)]` or similar attribute to restrict it to callers in the team lead role. Only looks like we do that in video web though